src/common/Logging/Log.cpp

```diff
@@ -356,8 +364,14 @@ Log* Log::instance()
     return &instance;
 }
 
-void Log::Initialize()
+void Log::Initialize(Acore::Asio::IoContext* ioContext)
 {
+    if (ioContext)
+    {
+        _ioContext = ioContext;
+        _strand = new Acore::Asio::Strand(*ioContext);
```

Comment:
It's generally a C++ code smell to add a `new` without a `delete`. Consider whether this needs proper deallocation (perhaps in a dtor).